map_org_names.yaml

```diff
@@ -604,11 +609,12 @@ mappings:
   - ['^[[:space:]]*4paradigm[[:space:]]第四范式[[:space:]]*$','Beijing Wusi Chuangxiang Technology Co., Ltd.']
   - ['^[[:space:]]*kuaishou[[:space:]]*$','BEIJING YUANDING TECHNOLOGY CO.,LTD']
   - ['^[[:space:]]*isure[[:space:]]cloud[[:space:]]*$','Beijing Yunyun Zhidu Technology Co., Ltd.']
-  - ['^[[:space:]]*&yet[[:space:]]*$','Beijing Zhiren Tianxia Technology Co., Ltd.']
+  - ['^[[:space:]]*//////[[:space:]]*$','Beijing Zhiren Tianxia Technology Co., Ltd.']
```

Comment:
Is this `//////` intended? What is it supposed to catch?

Reply:
yes, we have an org with the exact name `//////` in DA

```
id   |name  |src|op|
-----|------|---|--|
32224|//////|   |  |
```